src/wrap/wrap_util.c

```diff
@@ -43,16 +43,16 @@ char* SIZE_T_D;
 int
 to_coord_t(
         const char* const name,
-        PyObject* o,
-        coord_t* const c) {
+        PyArrayObject* o,
+        void * const c) {
 
-    PyObject* array = NULL;
+    PyArrayObject* array = NULL;
 
     if (o == NULL || o == Py_None) {
         return 0;
     }
 
-    array = PyArray_FromObject(o, NPY_DOUBLE, 1, 1);
+    array = (PyArrayObject *) PyArray_FromObject((PyObject *) o, NPY_DOUBLE, 1, 1);
```

Comment:
`(PyObject *) o` would not be needed if you would not make the change on 46. Why was that one needed?

Reply:
All of these pointless casts are left over from an attempt to solve the issue without changing the function signatures. I'm getting rid of them